components/sup/src/package/hooks.rs

```diff
@@ -16,18 +16,17 @@ use std::fmt;
 use std::fs::{self, File};
 use std::io::prelude::*;
 use std::path::PathBuf;
-use std::process::{Command, Stdio};
 
 use handlebars::Handlebars;
 
 use error::{Error, Result};
 use hcore::util;
-use hcore::os::users;
 use package::Package;
 use manager::service::config::{ServiceConfig, never_escape_fn};
 use util::convert;
 use util::handlebars_helpers;
 use util::users as hab_users;
+use util::create_command;
```

Comment:
Rust import guidelines lean against importing individual functions.  This could be `use util` with `util::create_comand` being called below.